typescript/utils/src/objects.ts

```diff
@@ -0,0 +1,13 @@
+import { ChainMap } from '@hyperlane-xyz/sdk';
+
+export function filteredOwners(
```

Comment:
Sorry, I should have been more clear. I do think this util function can live here but let's rename the vars to be something more generic